data/interim/korea/KOR_health.csv

```diff
@@ -1,1021 +1,1140 @@
-Seoul,2020-02-11,KOR,10,0,2
-Seoul,2020-02-12,KOR,10,0,2
-Seoul,2020-02-13,KOR,10,0,2
-Seoul,2020-02-14,KOR,10,0,2
-Seoul,2020-02-15,KOR,10,0,2
-Seoul,2020-02-16,KOR,12,0,3
-Seoul,2020-02-17,KOR,12,0,3
-Seoul,2020-02-18,KOR,12,0,3
-Seoul,2020-02-19,KOR,14,0,5
-Seoul,2020-02-20,KOR,18,0,5
-Seoul,2020-02-21,KOR,20,0,6
-Seoul,2020-02-22,KOR,24,0,6
-Seoul,2020-02-23,KOR,28,0,6
-Seoul,2020-02-24,KOR,32,0,6
-Seoul,2020-02-25,KOR,45,0,9
-Seoul,2020-02-26,KOR,55,0,9
-Seoul,2020-02-27,KOR,66,0,9
-Seoul,2020-02-28,KOR,73,0,9
-Seoul,2020-02-29,KOR,78,0,9
-Seoul,2020-03-01,KOR,84,0,9
-Seoul,2020-03-02,KOR,90,0,13
-Seoul,2020-03-03,KOR,91,0,15
-Seoul,2020-03-04,KOR,96,0,26
-Seoul,2020-03-05,KOR,97,0,26
-Seoul,2020-03-06,KOR,102,0,26
-Seoul,2020-03-07,KOR,108,0,27
-Seoul,2020-03-08,KOR,117,0,28
-Seoul,2020-03-09,KOR,139,0,29
-Seoul,2020-03-10,KOR,185,0,31
-Seoul,2020-03-11,KOR,202,0,37
-Seoul,2020-03-12,KOR,219,0,38
-Seoul,2020-03-13,KOR,231,0,40
-Seoul,2020-03-14,KOR,240,0,44
-Seoul,2020-03-15,KOR,248,0,47
-Seoul,2020-03-16,KOR,254,0,52
-Seoul,2020-03-17,KOR,267,0,55
-Seoul,2020-03-18,KOR,270,0,55
-Seoul,2020-03-19,KOR,282,0,58
-Ulsan,2020-01-20,KOR,0,0,0
-Ulsan,2020-01-21,KOR,0,0,0
-Ulsan,2020-01-22,KOR,0,0,0
-Ulsan,2020-01-23,KOR,0,0,0
-Ulsan,2020-01-24,KOR,0,0,0
-Ulsan,2020-01-25,KOR,0,0,0
-Ulsan,2020-01-26,KOR,0,0,0
-Ulsan,2020-01-27,KOR,0,0,0
-Ulsan,2020-01-28,KOR,0,0,0
-Ulsan,2020-01-29,KOR,0,0,0
-Ulsan,2020-01-30,KOR,0,0,0
-Ulsan,2020-01-31,KOR,0,0,0
-Ulsan,2020-02-01,KOR,0,0,0
-Ulsan,2020-02-02,KOR,0,0,0
-Ulsan,2020-02-03,KOR,0,0,0
-Ulsan,2020-02-04,KOR,0,0,0
-Ulsan,2020-02-05,KOR,0,0,0
-Ulsan,2020-02-06,KOR,0,0,0
-Ulsan,2020-02-07,KOR,0,0,0
-Ulsan,2020-02-08,KOR,0,0,0
-Ulsan,2020-02-09,KOR,0,0,0
-Ulsan,2020-02-10,KOR,0,0,0
-Ulsan,2020-02-11,KOR,0,0,0
-Ulsan,2020-02-12,KOR,0,0,0
-Ulsan,2020-02-13,KOR,0,0,0
-Ulsan,2020-02-14,KOR,0,0,0
-Ulsan,2020-02-15,KOR,0,0,0
-Ulsan,2020-02-16,KOR,0,0,0
-Ulsan,2020-02-17,KOR,0,0,0
-Ulsan,2020-02-18,KOR,0,0,0
-Ulsan,2020-02-19,KOR,0,0,0
-Ulsan,2020-02-20,KOR,0,0,0
-Ulsan,2020-02-21,KOR,0,0,0
-Ulsan,2020-02-22,KOR,1,0,0
-Ulsan,2020-02-23,KOR,1,0,0
-Ulsan,2020-02-24,KOR,2,0,0
-Ulsan,2020-02-25,KOR,4,0,0
-Ulsan,2020-02-26,KOR,5,0,0
-Ulsan,2020-02-27,KOR,11,0,0
-Ulsan,2020-02-28,KOR,17,0,0
-Ulsan,2020-02-29,KOR,17,0,0
-Ulsan,2020-03-01,KOR,20,0,0
-Ulsan,2020-03-02,KOR,20,0,0
-Ulsan,2020-03-03,KOR,23,0,0
-Ulsan,2020-03-04,KOR,23,0,0
-Ulsan,2020-03-05,KOR,23,0,1
-Ulsan,2020-03-06,KOR,23,0,1
-Ulsan,2020-03-07,KOR,24,0,1
-Ulsan,2020-03-08,KOR,24,0,1
-Ulsan,2020-03-09,KOR,24,0,1
-Ulsan,2020-03-10,KOR,25,0,1
-Ulsan,2020-03-11,KOR,25,0,2
-Ulsan,2020-03-12,KOR,27,0,3
-Ulsan,2020-03-13,KOR,27,0,3
-Ulsan,2020-03-14,KOR,28,0,5
-Ulsan,2020-03-15,KOR,28,0,5
-Ulsan,2020-03-16,KOR,28,0,6
-Ulsan,2020-03-17,KOR,30,0,7
-Ulsan,2020-03-18,KOR,36,0,7
-Ulsan,2020-03-19,KOR,36,0,9
\ No newline at end of file
+adm1_name,date,adm0_name,cumulative_confirmed_cases,cumulative_deaths,cumulative_recoveries,cumulative_active_cases

```

Comment:
Not that this really matters since it's not a processed file, but do we want to stick with the syntax of `cum_` vs `cumulative_`?

Also, what are "cumulative active cases". Should this just be "active cases"?

Reply:
oh! we drop and rename these later in our processing script, but I've renamed these variables for clarity, thanks for flagging!

Reply:
OK! Either way is fine. But if the names of an interim file got changed, can we include the update to the script that generates this interim file in this PR as well?